certi_builder/builder.py

```diff
@@ -31,42 +31,32 @@ def generate_certificate(
     for name in names_list:
         image = Image.open(certificate_image_path)
         drawImage = ImageDraw.Draw(image)
+        underline = underlineit(drawImage, name, location, text_color, font)
+        bold = boldit(drawImage, name, location, text_color)
+        writing = drawImage.text(location, name, fill=text_color, font=font)
```

Comment:
Hi, you don't need to assign a function call to a variable because it won't work, so it's better is to call a function when the condition is met.

Reply:
changes applied